mapper/mapper_test.go

```diff
@@ -27,14 +27,15 @@ func TestUnitMapper(t *testing.T) {
 			Title: "Cookies",
 		},
 	}
+	expectedModel.Language = "en"
 	expectedModel.Metadata.Title = "Cookies"
 	expectedModel.CookiesPreferencesSet = true
 	expectedModel.CookiesPolicy.Essential = true
 	expectedModel.CookiesPolicy.Usage = false
 	expectedModel.PreferencesUpdated = false
 	expectedModel.FeatureFlags.HideCookieBanner = true
 	Convey("test CreateCookieSettingPage", t, func() {
-		mcp := CreateCookieSettingPage(cookiesPolicy, false)
+		mcp := CreateCookieSettingPage(cookiesPolicy, false, "en")
```

Comment:
As we expect this to default to the default lang (although is and probably will always be "en") we should prob explicitly state that it is the default lang we expect it to be set to via dp-net/request/locale.DefaultLang